### DISCUSSION TITLE
How do you assert a List?

### DISCUSSION THREAD

**Discussion by sykesbPragmatics** [thomhurst/TUnit#1292] [Q&A]
We are trying to convert from NUnit and we are attempting to Assert that all elements in a list have a value:
```
Assert.That(results0.All(phone => phone == "(703) 890-8500"));
````

What is the "correct" way to do the same thing with TUnit?  I can't seem to find any examples dealing with lists.

Thanks!

**Comment by thomhurst** [comment]
`Assert.That(results0.All(phone => phone == "(703) 890-8500"));`

The equivalent in TUnit would be:

`Assert.That(results0.All(phone => phone == "(703) 890-8500")).IsTrue();`
